openjdk.test.mauve/build.xml

```diff
@@ -359,18 +359,43 @@ limitations under the License.
 		<delete dir="${openjdk_test_mauve_src_dir}/gnu/testlet/org/omg/PortableServer/POAOperations"/>
 	</target>
 
-	<target name="get-source" depends="download-cvsclient, create-work-dir" unless="openjdk_test_mauve_already_built">
-		<exec dir="${openjdk_test_mauve_work_dir}" executable="java">
-		<arg value="-Dcvs.root=:pserver:anoncvs@sourceware.org:/cvs/mauve"/>
-		<arg value="-jar"/>
-		<arg value="${first_prereqs_root}/cvsclient/org-netbeans-lib-cvsclient.jar"/>
-		<arg value="export"/>
-		<arg value="-r"/>
-		<arg value="HEAD"/>
-		<arg value="mauve"/>
-</exec>
+	<condition property="isJavaHomeSet" value="true">
+	   <isset property="JAVA_HOME"/>
+    </condition>
+
+    <condition property="isTestJDKHomeSet" value="true">
+	  <isset property="TEST_JDK_HOME"/>
+    </condition>
+
+    <target name="get-source-using-java-home" if="isJavaHomeSet".........> 
+        <exec dir="${openjdk_test_mauve_work_dir}" executable="java">
+		      <arg value="-Dcvs.root=:pserver:anoncvs@sourceware.org:/cvs/mauve"/>
+		      <arg value="-jar"/>
+		      <arg value="${first_prereqs_root}/cvsclient/org-netbeans-lib-cvsclient.jar"/>
+		      <arg value="export"/>
+		      <arg value="-r"/>
+		      <arg value="HEAD"/>
+		      <arg value="mauve"/>
+	    </exec>
 	</target>
 
+    <target name="get-source-using-test-jdk-home" if="isTestJDKHomeSet".........> 
```

Comment:
Same as above. The "..." portion should be removed.

Reply:
Once the PR is fixed, we need to run a Grinder to test this change. 

Here are some instructions on how to run grinder : https://github.com/adoptium/aqa-tests/wiki/How-to-Run-a-Grinder-Build-on-Jenkins . 

For this PR, you might specify the following: 

```
 - ADOPTOPENJDK_SYSTEMTEST_OWNER_BRANCH = SAY-droid427:mauve
 - BUILD_LIST = system 
 - TARGET = MauveSingleThrdLoad_J9_5m (or any other mauve test targets would suffice) 
 ```

You may accept default for all other grinder inputs. Please do let me know if you need any more information about how to test the PR.